packages/app/src/components/Form/Filters/OrderSubscriptions.tsx

```diff
@@ -0,0 +1,151 @@
+import { ResourceFinder } from '#components/Form/ResourceFinder'
+import {
+  InputDateRange,
+  InputSelect,
+  Spacer,
+  flatSelectValues,
+  useCoreSdkProvider
+} from '@commercelayer/app-elements'
+import {
+  type OrderSubscriptionField,
+  type OrderSubscriptionsFilters,
+  type FilterValue
+} from 'AppForm'
+import { useEffect, useState } from 'react'
+import { parseFilterToDate } from './utils'
+
+interface Props {
+  onChange: (filters: OrderSubscriptionsFilters) => void
+}
+
+export function OrderSubscriptions({ onChange }: Props): JSX.Element | null {
+  const { sdkClient } = useCoreSdkProvider()
+  const [filters, setFilter] = useState<OrderSubscriptionsFilters>({})
+
+  if (sdkClient == null) {
+    return null
+  }
+
+  const updateFilters = (
+    key: OrderSubscriptionField,
+    value: FilterValue
+  ): void => {
+    setFilter((state) => ({
+      ...state,
+      [key]: value
+    }))
+  }
+
+  useEffect(
+    function dispatchFilterChange() {
+      onChange(filters)
+    },
+    [filters]
+  )
+
+  return (
+    <div>
+      <Spacer bottom='6'>
+        <ResourceFinder
+          label='Markets'
+          resourceType='markets'
+          isMulti
+          onSelect={(values) => {
+            updateFilters('market_id_in', flatSelectValues(values))
+          }}
+          sdkClient={sdkClient}
+        />
+      </Spacer>
+
+      <Spacer bottom='6'>
+        <InputSelect
+          label='Status'
+          initialValues={[
+            {
+              value: 'draft',
+              label: 'Draft'
+            },
+            {
+              value: 'inactive',
+              label: 'Inactive'
+            },
+            {
+              value: 'active',
+              label: 'Active'
+            },
+            {
+              value: 'cancelled',
+              label: 'Cancelled'
+            }
+          ]}
+          isMulti
+          onSelect={(values) => {
+            updateFilters('status_in', flatSelectValues(values))
+          }}
+        />
+      </Spacer>
+
+      <Spacer bottom='6'>
+        <InputSelect
+          label='Frequency'
+          initialValues={[
+            {
+              value: 'hourly',
```

Comment:
here there is also the custom frequency with crontab... are we going to omit it?

Reply:
I was thinking of omitting the custom frequency to keep the UI simple.
Advanced users can still use custom filters.